src/ark/phenotyping/cell_cluster_utils.py

```diff
@@ -209,9 +209,9 @@ def add_consensus_labels_cell_table(base_dir, cell_table_path, cell_som_input_da
     # read in the data, ensure sorted by FOV column just in case
     cell_table = pd.read_csv(cell_table_path)
 
-    # for a simpler merge, rename segmentation_label to label in consensus_data
+    # for a simpler merge, rename label to label in consensus_data
     cell_som_results = cell_som_input_data.rename(
-        {'segmentation_label': 'label'}, axis=1
+        {'label': 'label'}, axis=1
```

Comment:
Is this necessary? Seems like you're renaming something to itself?

Reply:
spittin facts, I'll fix that.